resources/helm/overlays/istio/charts/tracing/templates/jaeger-all-in-one.yaml

```diff
@@ -19,19 +19,18 @@ spec:
       log-level: debug
       query:
         base-path: /
-    annotations:
```

Comment:
This is being removed without replacement, right?

Reply:
All of the components were using the same set of annotations - so rather than repeating on each component, thought it would be easier to just define them at the top level - and then put a guard around in case no annotations are actually specified?

Reply:
I see now, it does makes sense!